### ISSUE TITLE
team_code/orion_b2d_agent.py has code bug when using CARLA 0.9.15.

### ISSUE THREAD

**Issue by ziming-liu** [xiaomi-mlab/Orion#39]
According to recent Bench2drive benchmark, using carla 0.9.15, the agent API is `near_node, near_command = self._route_planner.run_step(pos)_**`. 

However, orion agent .py has ` (_, curr_command), (near_node, near_command) = self._route_planner.run_step(pos)`.  

I think the Bench2drive API has changed ? 

**_# (_, curr_command), (near_node, near_command) = self._route_planner.run_step(pos)
        near_node, near_command = self._route_planner.run_step(pos)_**

        if (math.isnan(compass) == True): #It can happen that the compass sends nan for a few frames
            compass = 0.0
            acceleration = np.zeros(3)
            angular_velocity = np.zeros(3)

        result = {
                'imgs': imgs,
                'gps': gps,
                'pos':pos,
                'speed': speed,
                'compass': compass,
                'bev': bev,
                'acceleration':acceleration,
                'angular_velocity':angular_velocity,
                # 'command_curr':curr_command,
                'command_near':near_command,
                'command_near_xy':near_node
                }
        
        return result
    
    @torch.no_grad()
    def run_step(self, input_data, timestamp):
        if not self.initialized:
            self._init()
        tick_data = self.tick(input_data)
        results = {}
        results['lidar2img'] = []
        results['lidar2cam'] = []
        results['cam_intrinsic'] = []
        results['img'] = []
        results['folder'] = ' '
        results['scene_token'] = ' '  
        results['frame_idx'] = self.step
        results['timestamp'] = self.step / 20
        results['box_type_3d'], _ = get_box_type('LiDAR')
  
        for cam in ['CAM_FRONT','CAM_FRONT_LEFT','CAM_FRONT_RIGHT','CAM_BACK','CAM_BACK_LEFT','CAM_BACK_RIGHT']:
            results['lidar2img'].append(self.lidar2img[cam])
            results['lidar2cam'].append(self.lidar2cam[cam])
            results['cam_intrinsic'].append(np.matmul(self.lidar2img[cam], np.linalg.inv(self.lidar2cam[cam])))
            results['img'].append(tick_data['imgs'][cam])
        results['lidar2img'] = np.stack(results['lidar2img'],axis=0)
        results['lidar2cam'] = np.stack(results['lidar2cam'],axis=0)
        raw_theta = tick_data['compass']   if not np.isnan(tick_data['compass']) else 0
        ego_theta = -raw_theta + np.pi/2
        rotation = list(Quaternion(axis=[0, 0, 1], radians=ego_theta))
        can_bus = np.zeros(18)
        can_bus[0] = tick_data['pos'][0]
        can_bus[1] = -tick_data['pos'][1]
        can_bus[3:7] = rotation
        can_bus[7] = tick_data['speed']
        can_bus[10:13] = tick_data['acceleration']
        can_bus[11] *= -1
        can_bus[13:16] = -tick_data['angular_velocity']
        can_bus[16] = ego_theta
        can_bus[17] = ego_theta / np.pi * 180 
        results['can_bus'] = can_bus
        **_# command = tick_data['command_curr']
        command = tick_data['command_near']
        # results['command'] = command2nohot(tick_data['command_curr'])
        # results['ego_fut_cmd'] = command2hot(tick_data['command_curr'])
        results['command'] = command2nohot(tick_data['command_near'])
        results['ego_fut_cmd'] = command2hot(tick_data['command_near'])_**

**Comment by wzh506** [comment]
You sholud use team_code/planner.py under Orion instead of Bench2Drive's planner.py or Bench2DriveZoo's planner.py. Maybe author has forgetted to inform this change although they have uploaded the team_code file.
